### DISCUSSION TITLE
[Opinion] Limiting reporting issues to contributors

### DISCUSSION THREAD

**Discussion by mdziczkowski** [community/community#42430] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

I think that possibility to limit avability to report issues to only contributors is a bery bad practice that it becomes missused, preventing other from reporting found problems to the software, with creares a potential risk for all those who would download it, not aware of them.

In my opinion, this feature should be removed and all project that had it enabled should have it reversed, to allow a normal use of it

**Comment by Poopooracoocoo** [comment]
Also see https://github.com/dear-github/dear-github/issues/293

I don't agree at all. GitHub issues needs more controls! Many maintainers want to limit who can create issues and who can create discussions. A lot want only discussions to be the place for users to report issues or receive support. This would make actual development significantly easier and reduces the burden from having to triage issues. Many don't get paid to do it so it's just consuming time and causes significant stress. This is the worst aspect of open source. There are ways to work around it, but such things remove the "Hub" in "GitHub". This is the biggest problem with issues and discussions at the moment. Too many support requests and incomplete bug reports and crazy suggestions and not enough contributions. There really needs to be a clear separation between contributors and users in GitHub itself. I've specifically got the ExplorerPatcher repository in mind. I've seen the toll the burden of triaging issues took on the maintainer. I've also got the GDLauncher and Ruffle repositories in mind, as well as a handful of others. Microsoft may be able to afford to hire dedicated issue triagers, but other companies and FOSS orgs and individuals can't.

**Reply by mdziczkowski** [reply]
The problem is that in many projects neither the discussions, wiki's nor else are enabled and the only way to contact the project team is thru the joined unde the "bug repot" board, where are both, issues and feature requests placed.

**Reply by Poopooracoocoo** [reply]
If someone doesn't want people to contact them about their project, then so be it. Source-available is equally right. People should not be forced to take on the burden of triaging or interacting with others if they just want to have their source on GitHub and want to use Issues to manage the development of their project.

But as it is now, it's either _everything_ for _everyone_, or _nothing_ for everyone. Developers have too much spam and poorly written issues on their hands as is. Starting with who can create issues means that it's not so black and white and empowers developers.
